lib/mayaUsd/ufe/UsdShaderNodeDef.cpp

```diff
@@ -48,6 +48,9 @@ Ufe::ConstAttributeDefs getAttrs(const PXR_NS::SdrShaderNodeConstPtr& shaderNode
     for (const PXR_NS::TfToken& name : names) {
         PXR_NS::SdrShaderPropertyConstPtr property
             = input ? shaderNodeDef->GetShaderInput(name) : shaderNodeDef->GetShaderOutput(name);
+        if (!property) {
+            continue;
```

Comment:
I have third-party shaders crashing here. They registered the input names, but did not register a SdrShaderProperty.

Reply:
That would be a great code comment.